infra/templates/az-isolated-service-single-region/docs/design.md

```diff
@@ -0,0 +1,169 @@
+# Deploying App Service into ASE 
```

Comment:
Comments related to the `template_topology.png`. You may want to consider including separate blue boxes for things like app services, webhooks, access restrictions, role assignments, etc. 
- `service_plan` - add a mention for ASE integration.
- `app_service` - add mentions for the staging slot, access restriction and webhook(s).
- `container_registry` - add a mention for image build + push
- rename `blob_storage`->`storage_account` - can you add a mention that we're creating both the storage account and container resources.

Reply:
Great feedback, I've updated the diagram